### ISSUE TITLE
Project Wiki Pages

### ISSUE THREAD

**Issue by ProGamerGov** [martinbenson/deep-photo-styletransfer#35]
I have created a basic Wiki for this project that links to the [Neural-Style Wiki](https://github.com/jcjohnson/neural-style/wiki), and pages on things related specifically to the deep-photo-styletransfer such as "How to create mask images for noobs", and "Mask related issues one might experience, and how to fix them".

The Wiki can be accessed normally through the Github menus, or through this link: 

https://github.com/martinbenson/deep-photo-styletransfer/wiki

It would be nice if others can add mask creation guides for the software and/or tools that they like to use, so that individuals can have a choice in how they create their mask images. There are also probably things that I have missed, and that are not covered on the Neural-Style Wiki, and thus it may be a good idea to add them to the deep-photo-styletransfer wiki. Feel free to also fix any errors that I may have accidentally made.

**Comment by tarvos21** [comment]
Wonderful work, thanks a lot! Really helpful to create my own masks.

**Comment by tarvos21** [comment]
Hi, @ProGamerGov, could I ask you how you install the G'MIC plug-in into GIMP? I am using a MacBook  and have installed GIMP 2.8.20, but I am not able to get the G'MIC plug-in to work well. I tried to copy the files into GIMP plugin folder, and Ctrl+open it to grant system the access to open it. But in GIMP filters, there is no G'MIC options.

**Comment by tarvos21** [comment]
After trial and error for hours, I suggest that, for those who wants to creat their own masks on macOS Sierra, it's better to install GIMP from this resource(GIMPonOSX):
http://gimp.lisanet.de/Website/Download.html

and GMIC plugin from here(official):
http://karo03.bplaced.de/gmic/release/

thus the GMIC plugin will work normally.

The possible plug-ins directory:
`/Applications/GIMP.app/Contents/Resources/lib/gimp/2.0/plug-ins`
`/Users/{your-usename}/Library/Applications Support/GIMP/2.8/plug-ins/`

Check which works for you.

Don't forget to install Xquartz to make it work.

Thanks for the WIKI!

**Comment by ProGamerGov** [comment]
@tarvos21 I installed the plug-in on Windows by running the `gmic_gimp_win64.exe` installation file. Glad to see you figure out the issue! 

I have also added 2 new guides to the Wiki: 

https://github.com/martinbenson/deep-photo-styletransfer/wiki/How-To-Create-Dilated-Versions-Of-Your-Masks

https://github.com/martinbenson/deep-photo-styletransfer/wiki/Creating-Mask-Regions-With-GIMP's-Intelligent-Scissors-Tool


I've created these guides based on what I have learned/figured out myself. So there may be more efficient ways of doing things. But the main purpose of the guides is to lower to barrier to entry with style transfer projects that require mask images. 

Let me know if I'm missing anything regarding mask image creation with these guides.
